### ISSUE TITLE
Add "Philosophy" area (aka. type)

### ISSUE THREAD

**Issue by redbar0n** [ybogdanov/history-timeline#37]
Some philosophers are listed under "Science", but Philosophy should have its own area I think. I would like to map out the philosophers from here https://www.denizcemonduygu.com/philo/browse/

**Comment by fishenal** [comment]
@redbar0n this repo seems not in maintenance, do you interesting to create a new one(or fork this one)? I can help to contribute.

**Comment by redbar0n** [comment]
@fishenal indeed, but unfortunately I can't afford spending time on it at this point.
